libbeat/docs/processors-using.asciidoc

```diff
@@ -469,7 +469,7 @@ exist in the event are overwritten by keys from the decoded JSON object. The
 default value is false.
 
 [[drop-event]]
-=== Drop events
+=== Drop event
```

Comment:
I would either say "Drop an event" or "Drop events" (for consistency with other docs).

Reply:
Drop fields is actually plural because it can drop multiple fields. As the processor only processes one event, it can only drop one at the time and is why the processor name is singular.

Let me know if you prefer to keep as is.

Reply:
I see where you're coming from (the processor itself drops a single event at at time, unlike `drop_fields`, which can drop multiple fields). But from the user's perspective, won't the processor drop all events that match the specified condition? In general, we should use titles that describe the user goal rather than how the software behaves.